saleor/checkout/models.py

```diff
@@ -86,6 +86,7 @@ class Checkout(models.Model):
         related_name="checkouts",
         on_delete=models.SET_NULL,
     )
+    customer_note = models.TextField(blank=True, default="")
```

Comment:
I'm not sure if we should add a new field to model. IMO changing the API is enough

Reply:
I agree with @IKarbowiak. If we want to indicate that `note` is a `customer_note` in the API, we could leave a comment above the `note` field.